OMCompiler/Compiler/runtime/OMSimulator_omc.c

```diff
@@ -826,6 +845,17 @@ extern const int OMSimulator_oms_importFile(const char* filename, char** cref)
   return status;
 }
 
+extern const int OMSimulator_oms_importSnapshot(const char* cref, const char* snapshot)
+{
+  if(!oms_importSnapshot)
+  {
+    printf("Could not Locate the Function oms_importSnapshot\n");
```

Comment:
Please fix upper and lower case in this and the other messages.

Reply:
fixed with https://github.com/OpenModelica/OpenModelica/pull/6831/commits/d2ce5540c6c3e12d50a447d3bc53e9aa98851f55